charter.md

```diff
@@ -20,20 +20,20 @@ previous Secure Shell WG has been closed and its documents published. In the
 intervening years, current day implementations and RFCs have diverged, although
 the (now closed) curdle WG and various AD-sponsored RFCs have updated some of
 the SSH documentation, but in a somewhat sporadic manner as there was no WG
-chartered to generally maintain the SSH protocol documentation. The first goal
-of this new working group is to update the RFCs documenting SSH to reflect what
-is implemented and deployed in practice. In particular, the working group will
-document the OpenSSH certificate structure, the SSH agent protocol, and SFTP,
-as they are commonly implemented.
-
-A second goal of this working group is to update and maintain the list of
-cryptographic algorithms used by SSH. This includes documenting existing
-algorithms, deprecating unsafe algorithms, defining new algorithms (such as
-post-quantum), and determining the set of recommended and
-mandatory-to-implement algorithms. Updating IANA SSH registries and changing
-their registration policies is in scope.
-
-While the development of formal verification proofs is out of scope, this
+chartered to generally maintain the SSH protocol documentation. 
+
+The initial goals of this new working group are:   
+
+* to update the RFCs documenting SSH to reflect what is implemented and deployed in practice.
+  In particular, the working group will document the OpenSSH certificate structure, the SSH agent
+  protocol, and SFTP, as they are commonly implemented.
+
+* to update and maintain the list of cryptographic algorithms used by SSH. This includes documenting
+  existing algorithms, deprecating unsafe algorithms, defining new algorithms (such as post-quantum),
```

Comment:
Perhaps “selecting” instead of “defining”?

Reply:
yes, that is better.  TY

Reply:
If you change the word and commit again, and push again, it should show up in this PR

Reply:
Ha!  I think I did it.